runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -1322,7 +1322,8 @@ private void addSwaps()
 
 		if (this.swapCoalBag)
 		{
-			menuManager.addPriorityEntry("Empty", "Coal bag");
+			menuManager.addPriorityEntry("Empty", "Coal bag").setPriority(90);
+			menuManager.addPriorityEntry("Fill", "Coal bag").setPriority(100);
```

Comment:
aren't these the wrong way around?

Reply:
> 
> 
> aren't these the wrong way around?

Pretty sure you'd rather fill the bag first than empty it.

Reply:
Fill is not a menu entry when the bank interface is open. This makes it so that left click is fill outside of the bank and left click is empty inside of the bank. Unless I'm mistaken, that's what it's supposed to do.

Reply:
BankComparableEntry is definitely better but this was low effort.

Reply:
> 
> 
> BankComparableEntry is definitely better but this was low effort.

Bank comparable entry is just adding new bankcomparable entry before the "fill", "coal bag"